### ISSUE TITLE
reg loss function

### ISSUE THREAD

**Issue by leap4133** [zylo117/Yet-Another-EfficientDet-Pytorch#170]
When I go through your code, I noticed that you use the cut-off threshold 1/9 for smoothed L1 loss. I was wondering why we use such kind of cutoff.

**Comment by Eckert-ZJB** [comment]
I also noticed this, but I don't know why. By the way, when training my own data, the reg loss is very small at the begining. In fact, It has been oscillating throughout the training process, even though the total loss seems to converge. Why I got this result? @zylo117 
![image](https://user-images.githubusercontent.com/50358780/83127196-72726300-a10c-11ea-91fa-ae349bb3355e.png)
![image](https://user-images.githubusercontent.com/50358780/83127243-7f8f5200-a10c-11ea-8193-1bc537764d95.png)
